### ISSUE TITLE
ValueError: No escaped character

### ISSUE THREAD

**Issue by alecastle** [nvbn/thefuck#908]
The output of `thefuck --version` (something like `The Fuck 3.1 using Python
3.5.0 and Bash 4.4.12(1)-release`):

    The Fuck 3.28 using Python 3.7.3 and Bash 5.0.3(1)-release

Your system (Debian 7, ArchLinux, Windows, etc.):

    Arch Linux

How to reproduce the bug:

call the program after trying to run `sudo pacman -s atom` (note the lack of capitalization)

The output of The Fuck with `THEFUCK_DEBUG=true` exported (typically execute `export THEFUCK_DEBUG=true` in your shell before The Fuck):

```
DEBUG: Run with settings: {'alter_history': True,
 'debug': True,
 'env': {'GIT_TRACE': '1', 'LANG': 'C', 'LC_ALL': 'C'},
 'exclude_rules': [],
 'history_limit': None,
 'instant_mode': False,
 'no_colors': False,
 'num_close_matches': 3,
 'priority': {},
 'repeat': False,
 'require_confirmation': True,
 'rules': [<const: All rules enabled>],
 'slow_commands': ['lein', 'react-native', 'gradle', './gradlew', 'vagrant'],
 'user_dir': PosixPath('/home/alecastle/.config/thefuck'),
 'wait_command': 3,
 'wait_slow_command': 15}
DEBUG: Received output: error: invalid option '-s'

DEBUG: Call: sudo pacman -s atom; with env: {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/alecastle:@/tmp/.ICE-unix/1720,unix/alecastle:/tmp/.ICE-unix/1720', 'TF_SHELL': 'bash', 'COLORTERM': 'truecolor', 'XDG_CONFIG_DIRS': '/etc/xdg', 'XDG_SESSION_PATH': '/org/freedesktop/DisplayManager/Session1', 'XDG_MENU_PREFIX': 'xfce-', 'SSH_AUTH_SOCK': '/tmp/ssh-DYm70ErbnToU/agent.1740', 'DESKTOP_SESSION': '/usr/share/xsessions/xfce', 'SSH_AGENT_PID': '1741', 'EDITOR': 'vim', 'GTK_MODULES': 'canberra-gtk-module:canberra-gtk-module', 'XDG_SEAT': 'seat0', 'PWD': '/home/alecastle/Desktop', 'LOGNAME': 'alecastle', 'XDG_SESSION_DESKTOP': 'XFCE', 'XDG_SESSION_TYPE': 'x11', 'XAUTHORITY': '/home/alecastle/.Xauthority', 'HOME': '/home/alecastle', 'LANG': 'C', 'XDG_CURRENT_DESKTOP': 'XFCE', 'VTE_VERSION': '5403', 'XDG_SEAT_PATH': '/org/freedesktop/DisplayManager/Seat0', 'GLADE_CATALOG_PATH': ':', 'TF_ALIAS': 'fuck', 'XDG_SESSION_CLASS': 'user', 'TERM': 'xterm-256color', 'TF_HISTORY': '\t fuck\n\t fuck\n\t source ~/.bashrc\n\t pacman -s atom\n\t fuck\n\t*sudo pacman -s atom\\\n\t fuck\n\t thefuck --version\n\t export THEFUCK_DEBUG=true\n\t sudo pacman -s atom', 'USER': 'alecastle', 'PYTHONIOENCODING': 'utf-8', 'TF_SHELL_ALIASES': "alias ls='ls --color=auto'", 'DISPLAY': ':0.0', 'SHLVL': '1', 'XDG_VTNR': '1', 'XDG_SESSION_ID': '2', 'MOZ_PLUGIN_PATH': '/usr/lib/mozilla/plugins', 'GLADE_MODULE_PATH': ':', 'XDG_RUNTIME_DIR': '/run/user/1000', 'GLADE_PIXMAP_PATH': ':', 'XDG_DATA_DIRS': '/usr/local/share:/usr/share', 'GDK_BACKEND': 'x11', 'PATH': '/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl', 'THEFUCK_DEBUG': 'true', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1000/bus', 'MAIL': '/var/spool/mail/alecastle', 'OLDPWD': '/home/alecastle', '_': '/usr/bin/thefuck', 'LC_ALL': 'C', 'GIT_TRACE': '1'}; is slow:  took: 0:00:00.028039
DEBUG: Importing rule: adb_unknown_command; took: 0:00:00.000522
DEBUG: Importing rule: ag_literal; took: 0:00:00.000472
DEBUG: Importing rule: apt_get; took: 0:00:00.000637
DEBUG: Importing rule: apt_get_search; took: 0:00:00.000323
DEBUG: Importing rule: apt_invalid_operation; took: 0:00:00.000747
DEBUG: Importing rule: apt_list_upgradable; took: 0:00:00.000396
DEBUG: Importing rule: apt_upgrade; took: 0:00:00.000398
DEBUG: Importing rule: aws_cli; took: 0:00:00.000365
DEBUG: Importing rule: az_cli; took: 0:00:00.000274
DEBUG: Importing rule: brew_cask_dependency; took: 0:00:00.000582
DEBUG: Importing rule: brew_install; took: 0:00:00.000118
DEBUG: Importing rule: brew_link; took: 0:00:00.000277
DEBUG: Importing rule: brew_reinstall; took: 0:00:00.000560
DEBUG: Importing rule: brew_uninstall; took: 0:00:00.000279
DEBUG: Importing rule: brew_unknown_command; took: 0:00:00.000134
DEBUG: Importing rule: brew_update_formula; took: 0:00:00.000277
DEBUG: Importing rule: cargo; took: 0:00:00.000094
DEBUG: Importing rule: cargo_no_command; took: 0:00:00.000254
DEBUG: Importing rule: cat_dir; took: 0:00:00.000259
DEBUG: Importing rule: cd_correction; took: 0:00:00.001061
DEBUG: Importing rule: cd_mkdir; took: 0:00:00.000382
DEBUG: Importing rule: cd_parent; took: 0:00:00.000097
DEBUG: Importing rule: chmod_x; took: 0:00:00.000093
DEBUG: Importing rule: composer_not_command; took: 0:00:00.000320
DEBUG: Importing rule: cp_omitting_directory; took: 0:00:00.000393
DEBUG: Importing rule: cpp11; took: 0:00:00.000254
DEBUG: Importing rule: dirty_untar; took: 0:00:00.001132
DEBUG: Importing rule: dirty_unzip; took: 0:00:00.000946
DEBUG: Importing rule: django_south_ghost; took: 0:00:00.000103
DEBUG: Importing rule: django_south_merge; took: 0:00:00.000088
DEBUG: Importing rule: dnf_no_such_command; took: 0:00:00.000700
DEBUG: Importing rule: docker_not_command; took: 0:00:00.000559
DEBUG: Importing rule: dry; took: 0:00:00.000101
DEBUG: Importing rule: fab_command_not_found; took: 0:00:00.000364
DEBUG: Importing rule: fix_alt_space; took: 0:00:00.000293
DEBUG: Importing rule: fix_file; took: 0:00:00.001952
DEBUG: Importing rule: gem_unknown_command; took: 0:00:00.000390
DEBUG: Importing rule: git_add; took: 0:00:00.000479
DEBUG: Importing rule: git_add_force; took: 0:00:00.000289
DEBUG: Importing rule: git_bisect_usage; took: 0:00:00.000261
DEBUG: Importing rule: git_branch_delete; took: 0:00:00.000248
DEBUG: Importing rule: git_branch_exists; took: 0:00:00.000333
DEBUG: Importing rule: git_branch_list; took: 0:00:00.000252
DEBUG: Importing rule: git_checkout; took: 0:00:00.000258
DEBUG: Importing rule: git_commit_amend; took: 0:00:00.000245
DEBUG: Importing rule: git_diff_no_index; took: 0:00:00.000259
DEBUG: Importing rule: git_diff_staged; took: 0:00:00.000250
DEBUG: Importing rule: git_fix_stash; took: 0:00:00.000262
DEBUG: Importing rule: git_flag_after_filename; took: 0:00:00.000271
DEBUG: Importing rule: git_help_aliased; took: 0:00:00.000283
DEBUG: Importing rule: git_merge; took: 0:00:00.000306
DEBUG: Importing rule: git_merge_unrelated; took: 0:00:00.000262
DEBUG: Importing rule: git_not_command; took: 0:00:00.000258
DEBUG: Importing rule: git_pull; took: 0:00:00.000253
DEBUG: Importing rule: git_pull_clone; took: 0:00:00.000257
DEBUG: Importing rule: git_pull_uncommitted_changes; took: 0:00:00.000248
DEBUG: Importing rule: git_push; took: 0:00:00.000287
DEBUG: Importing rule: git_push_different_branch_names; took: 0:00:00.000247
DEBUG: Importing rule: git_push_force; took: 0:00:00.000251
DEBUG: Importing rule: git_push_pull; took: 0:00:00.000250
DEBUG: Importing rule: git_push_without_commits; took: 0:00:00.000299
DEBUG: Importing rule: git_rebase_merge_dir; took: 0:00:00.000265
DEBUG: Importing rule: git_rebase_no_changes; took: 0:00:00.000187
DEBUG: Importing rule: git_remote_delete; took: 0:00:00.000247
DEBUG: Importing rule: git_remote_seturl_add; took: 0:00:00.000181
DEBUG: Importing rule: git_rm_local_modifications; took: 0:00:00.000250
DEBUG: Importing rule: git_rm_recursive; took: 0:00:00.000247
DEBUG: Importing rule: git_rm_staged; took: 0:00:00.000253
DEBUG: Importing rule: git_stash; took: 0:00:00.000248
DEBUG: Importing rule: git_stash_pop; took: 0:00:00.000251
DEBUG: Importing rule: git_tag_force; took: 0:00:00.000242
DEBUG: Importing rule: git_two_dashes; took: 0:00:00.000250
DEBUG: Importing rule: go_run; took: 0:00:00.000268
DEBUG: Importing rule: gradle_no_task; took: 0:00:00.000514
DEBUG: Importing rule: gradle_wrapper; took: 0:00:00.000265
DEBUG: Importing rule: grep_arguments_order; took: 0:00:00.000266
DEBUG: Importing rule: grep_recursive; took: 0:00:00.000258
DEBUG: Importing rule: grunt_task_not_found; took: 0:00:00.000447
DEBUG: Importing rule: gulp_not_task; took: 0:00:00.000287
DEBUG: Importing rule: has_exists_script; took: 0:00:00.000257
DEBUG: Importing rule: heroku_multiple_apps; took: 0:00:00.000266
DEBUG: Importing rule: heroku_not_command; took: 0:00:00.000253
DEBUG: Importing rule: history; took: 0:00:00.000101
DEBUG: Importing rule: hostscli; took: 0:00:00.000414
DEBUG: Importing rule: ifconfig_device_not_found; took: 0:00:00.000363
DEBUG: Importing rule: java; took: 0:00:00.000261
DEBUG: Importing rule: javac; took: 0:00:00.000263
DEBUG: Importing rule: lein_not_task; took: 0:00:00.000420
DEBUG: Importing rule: ln_no_hard_link; took: 0:00:00.000250
DEBUG: Importing rule: ln_s_order; took: 0:00:00.000301
DEBUG: Importing rule: long_form_help; took: 0:00:00.000101
DEBUG: Importing rule: ls_all; took: 0:00:00.000269
DEBUG: Importing rule: ls_lah; took: 0:00:00.000292
DEBUG: Importing rule: man; took: 0:00:00.000280
DEBUG: Importing rule: man_no_space; took: 0:00:00.000094
DEBUG: Importing rule: mercurial; took: 0:00:00.000258
DEBUG: Importing rule: missing_space_before_subcommand; took: 0:00:00.000107
DEBUG: Importing rule: mkdir_p; took: 0:00:00.000257
DEBUG: Importing rule: mvn_no_command; took: 0:00:00.000262
DEBUG: Importing rule: mvn_unknown_lifecycle_phase; took: 0:00:00.000263
DEBUG: Importing rule: no_command; took: 0:00:00.000282
DEBUG: Importing rule: no_such_file; took: 0:00:00.000101
DEBUG: Importing rule: npm_missing_script; took: 0:00:00.000508
DEBUG: Importing rule: npm_run_script; took: 0:00:00.000266
DEBUG: Importing rule: npm_wrong_command; took: 0:00:00.000475
DEBUG: Importing rule: open; took: 0:00:00.000355
DEBUG: Importing rule: pacman; took: 0:00:00.000339
DEBUG: Importing rule: pacman_not_found; took: 0:00:00.000103
DEBUG: Importing rule: path_from_history; took: 0:00:00.000121
DEBUG: Importing rule: php_s; took: 0:00:00.000600
DEBUG: Importing rule: pip_unknown_command; took: 0:00:00.000349
DEBUG: Importing rule: port_already_in_use; took: 0:00:00.000169
DEBUG: Importing rule: prove_recursively; took: 0:00:00.000265
DEBUG: Importing rule: python_command; took: 0:00:00.000257
DEBUG: Importing rule: python_execute; took: 0:00:00.000261
DEBUG: Importing rule: quotation_marks; took: 0:00:00.000139
DEBUG: Importing rule: react_native_command_unrecognized; took: 0:00:00.000336
DEBUG: Importing rule: remove_trailing_cedilla; took: 0:00:00.000100
DEBUG: Importing rule: rm_dir; took: 0:00:00.000262
DEBUG: Importing rule: rm_root; took: 0:00:00.000252
DEBUG: Importing rule: scm_correction; took: 0:00:00.000347
DEBUG: Importing rule: sed_unterminated_s; took: 0:00:00.000265
DEBUG: Importing rule: sl_ls; took: 0:00:00.000099
DEBUG: Importing rule: ssh_known_hosts; took: 0:00:00.000267
DEBUG: Importing rule: sudo; took: 0:00:00.000102
DEBUG: Importing rule: sudo_command_from_user_path; took: 0:00:00.000262
DEBUG: Importing rule: switch_lang; took: 0:00:00.000120
DEBUG: Importing rule: systemctl; took: 0:00:00.000396
DEBUG: Importing rule: test.py; took: 0:00:00.000131
DEBUG: Importing rule: tmux; took: 0:00:00.000272
DEBUG: Importing rule: touch; took: 0:00:00.000342
DEBUG: Importing rule: tsuru_login; took: 0:00:00.000277
DEBUG: Importing rule: tsuru_not_command; took: 0:00:00.000270
DEBUG: Importing rule: unknown_command; took: 0:00:00.000111
DEBUG: Importing rule: unsudo; took: 0:00:00.000100
DEBUG: Importing rule: vagrant_up; took: 0:00:00.000271
DEBUG: Importing rule: whois; took: 0:00:00.000415
DEBUG: Importing rule: workon_doesnt_exists; took: 0:00:00.000354
DEBUG: Importing rule: yarn_alias; took: 0:00:00.000278
DEBUG: Importing rule: yarn_command_not_found; took: 0:00:00.000543
DEBUG: Importing rule: yarn_command_replaced; took: 0:00:00.000357
DEBUG: Importing rule: yarn_help; took: 0:00:00.000264
DEBUG: Trying rule: path_from_history; took: 0:00:00.000449
DEBUG: Trying rule: dry; took: 0:00:00.000056
DEBUG: Trying rule: git_stash_pop; took: 0:00:00.000017
DEBUG: Trying rule: test.py; took: 0:00:00.000001
DEBUG: Trying rule: adb_unknown_command; took: 0:00:00.000010
DEBUG: Trying rule: ag_literal; took: 0:00:00.000010
DEBUG: Trying rule: aws_cli; took: 0:00:00.000008
DEBUG: Trying rule: az_cli; took: 0:00:00.000008
DEBUG: Trying rule: brew_link; took: 0:00:00.000009
DEBUG: Trying rule: brew_reinstall; took: 0:00:00.000006
DEBUG: Trying rule: brew_uninstall; took: 0:00:00.000006
DEBUG: Trying rule: brew_update_formula; took: 0:00:00.000007
DEBUG: Trying rule: cargo; took: 0:00:00.000001
DEBUG: Trying rule: cargo_no_command; took: 0:00:00.000008
DEBUG: Trying rule: cat_dir; took: 0:00:00.000008
DEBUG: Trying rule: cd_correction; took: 0:00:00.000042
DEBUG: Trying rule: cd_mkdir; took: 0:00:00.000011
DEBUG: Trying rule: cd_parent; took: 0:00:00.000001
DEBUG: Trying rule: chmod_x; took: 0:00:00.000002
DEBUG: Trying rule: composer_not_command; took: 0:00:00.000010
DEBUG: Trying rule: cp_omitting_directory; took: 0:00:00.000037
DEBUG: Trying rule: cpp11; took: 0:00:00.000010
DEBUG: Trying rule: dirty_untar; took: 0:00:00.000008
DEBUG: Trying rule: dirty_unzip; took: 0:00:00.000008
DEBUG: Trying rule: django_south_ghost; took: 0:00:00.000002
DEBUG: Trying rule: django_south_merge; took: 0:00:00.000001
DEBUG: Trying rule: docker_not_command; took: 0:00:00.000036
DEBUG: Trying rule: fab_command_not_found; took: 0:00:00.000010
DEBUG: Trying rule: fix_alt_space; took: 0:00:00.000006
DEBUG: Trying rule: fix_file; took: 0:00:00.000015
DEBUG: Trying rule: gem_unknown_command; took: 0:00:00.000009
DEBUG: Trying rule: git_add; took: 0:00:00.000007
DEBUG: Trying rule: git_add_force; took: 0:00:00.000007
DEBUG: Trying rule: git_bisect_usage; took: 0:00:00.000006
DEBUG: Trying rule: git_branch_delete; took: 0:00:00.000006
DEBUG: Trying rule: git_branch_exists; took: 0:00:00.000006
DEBUG: Trying rule: git_branch_list; took: 0:00:00.000006
DEBUG: Trying rule: git_checkout; took: 0:00:00.000006
DEBUG: Trying rule: git_commit_amend; took: 0:00:00.000006
DEBUG: Trying rule: git_diff_no_index; took: 0:00:00.000006
DEBUG: Trying rule: git_diff_staged; took: 0:00:00.000006
DEBUG: Trying rule: git_fix_stash; took: 0:00:00.000006
DEBUG: Trying rule: git_flag_after_filename; took: 0:00:00.000005
DEBUG: Trying rule: git_help_aliased; took: 0:00:00.000006
DEBUG: Trying rule: git_merge; took: 0:00:00.000006
DEBUG: Trying rule: git_merge_unrelated; took: 0:00:00.000005
DEBUG: Trying rule: git_not_command; took: 0:00:00.000006
DEBUG: Trying rule: git_pull; took: 0:00:00.000006
DEBUG: Trying rule: git_pull_clone; took: 0:00:00.000006
DEBUG: Trying rule: git_pull_uncommitted_changes; took: 0:00:00.000009
DEBUG: Trying rule: git_push; took: 0:00:00.000006
DEBUG: Trying rule: git_push_different_branch_names; took: 0:00:00.000005
DEBUG: Trying rule: git_push_pull; took: 0:00:00.000005
DEBUG: Trying rule: git_push_without_commits; took: 0:00:00.000006
DEBUG: Trying rule: git_rebase_merge_dir; took: 0:00:00.000006
DEBUG: Trying rule: git_rebase_no_changes; took: 0:00:00.000006
DEBUG: Trying rule: git_remote_delete; took: 0:00:00.000006
DEBUG: Trying rule: git_remote_seturl_add; took: 0:00:00.000006
DEBUG: Trying rule: git_rm_local_modifications; took: 0:00:00.000008
DEBUG: Trying rule: git_rm_recursive; took: 0:00:00.000006
DEBUG: Trying rule: git_rm_staged; took: 0:00:00.000005
DEBUG: Trying rule: git_stash; took: 0:00:00.000006
DEBUG: Trying rule: git_tag_force; took: 0:00:00.000005
DEBUG: Trying rule: git_two_dashes; took: 0:00:00.000006
DEBUG: Trying rule: go_run; took: 0:00:00.000008
DEBUG: Trying rule: gradle_no_task; took: 0:00:00.000008
DEBUG: Trying rule: gradle_wrapper; took: 0:00:00.000008
DEBUG: Trying rule: grep_arguments_order; took: 0:00:00.000010
DEBUG: Trying rule: grep_recursive; took: 0:00:00.000008
DEBUG: Trying rule: grunt_task_not_found; took: 0:00:00.000007
DEBUG: Trying rule: gulp_not_task; took: 0:00:00.000008
DEBUG: Trying rule: has_exists_script; took: 0:00:00.000041
DEBUG: Trying rule: heroku_multiple_apps; took: 0:00:00.000027
DEBUG: Trying rule: heroku_not_command; took: 0:00:00.000008
DEBUG: Trying rule: hostscli; took: 0:00:00.000052
DEBUG: Trying rule: ifconfig_device_not_found; took: 0:00:00.000013
DEBUG: Trying rule: java; took: 0:00:00.000009
DEBUG: Trying rule: javac; took: 0:00:00.000008
DEBUG: Trying rule: lein_not_task; took: 0:00:00.000036
DEBUG: Trying rule: ln_no_hard_link; took: 0:00:00.000004
DEBUG: Trying rule: ln_s_order; took: 0:00:00.000027
DEBUG: Trying rule: ls_all; took: 0:00:00.000013
DEBUG: Trying rule: ls_lah; took: 0:00:00.000008
DEBUG: Trying rule: man; took: 0:00:00.000010
DEBUG: Trying rule: mercurial; took: 0:00:00.000008
DEBUG: Trying rule: mkdir_p; took: 0:00:00.000004
DEBUG: Trying rule: mvn_no_command; took: 0:00:00.000008
DEBUG: Trying rule: mvn_unknown_lifecycle_phase; took: 0:00:00.000006
DEBUG: Trying rule: no_such_file; took: 0:00:00.000532
DEBUG: Trying rule: npm_missing_script; took: 0:00:00.000014
DEBUG: Trying rule: npm_run_script; took: 0:00:00.000008
DEBUG: Trying rule: npm_wrong_command; took: 0:00:00.000044
DEBUG: Trying rule: open; took: 0:00:00.000011
DEBUG: Trying rule: php_s; took: 0:00:00.000012
DEBUG: Trying rule: pip_unknown_command; took: 0:00:00.000039
DEBUG: Trying rule: port_already_in_use; took: 0:00:00.000416
DEBUG: Trying rule: prove_recursively; took: 0:00:00.000013
DEBUG: Trying rule: python_command; took: 0:00:00.000036
DEBUG: Trying rule: python_execute; took: 0:00:00.000012
DEBUG: Trying rule: quotation_marks; took: 0:00:00.000001
DEBUG: Trying rule: react_native_command_unrecognized; took: 0:00:00.000009
DEBUG: Trying rule: remove_trailing_cedilla; took: 0:00:00.000002
DEBUG: Trying rule: rm_dir; took: 0:00:00.000005
DEBUG: Trying rule: scm_correction; took: 0:00:00.000013
DEBUG: Trying rule: sed_unterminated_s; took: 0:00:00.000008
DEBUG: Trying rule: sl_ls; took: 0:00:00.000001
DEBUG: Trying rule: ssh_known_hosts; took: 0:00:00.000008
DEBUG: Trying rule: sudo; took: 0:00:00.000002
DEBUG: Trying rule: sudo_command_from_user_path; took: 0:00:00.000009
DEBUG: Trying rule: switch_lang; took: 0:00:00.000001
DEBUG: Trying rule: systemctl; took: 0:00:00.000039
DEBUG: Trying rule: tmux; took: 0:00:00.000010
DEBUG: Trying rule: touch; took: 0:00:00.000008
DEBUG: Trying rule: tsuru_login; took: 0:00:00.000010
DEBUG: Trying rule: tsuru_not_command; took: 0:00:00.000007
DEBUG: Trying rule: unknown_command; took: 0:00:00.000103
DEBUG: Trying rule: unsudo; took: 0:00:00.000003
DEBUG: Trying rule: vagrant_up; took: 0:00:00.000011
DEBUG: Trying rule: whois; took: 0:00:00.000009
DEBUG: Trying rule: workon_doesnt_exists; took: 0:00:00.000008
DEBUG: Trying rule: yarn_alias; took: 0:00:00.000008
DEBUG: Trying rule: yarn_command_not_found; took: 0:00:00.000008
DEBUG: Trying rule: yarn_command_replaced; took: 0:00:00.000007
DEBUG: Trying rule: yarn_help; took: 0:00:00.000008
DEBUG: Trying rule: man_no_space; took: 0:00:00.000008
DEBUG: Trying rule: no_command; took: 0:00:00.000073
DEBUG: Trying rule: missing_space_before_subcommand; took: 0:00:00.000027
DEBUG: Trying rule: long_form_help; took: 0:00:00.000307
DEBUG: Trying rule: history; took: 0:00:00.002443
sudo pacman -S aviindex [enter/↑/↓/ctrl+c]DEBUG: Corrected commands: 
sudo pacman -S aviindex [enter/↑/↓/ctrl+c]
Aborted
DEBUG: Total took: 0:00:29.098330
```

If the bug only appears with a specific application, the output of that application and its version:

```
Guake Terminal: 3.5.0
VTE: 0.56.1
VTE runtime: 0.56.1
Gtk: 3.24.8
```

Anything else you think is relevant:

The original output before setting debug to true:
```
Traceback (most recent call last):
  File "/usr/bin/thefuck", line 12, in <module>
    sys.exit(main())
  File "/usr/lib/python3.7/site-packages/thefuck/entrypoints/main.py", line 26, in main
    fix_command(known_args)
  File "/usr/lib/python3.7/site-packages/thefuck/entrypoints/fix_command.py", line 36, in fix_command
    command = types.Command.from_raw_script(raw_command)
  File "/usr/lib/python3.7/site-packages/thefuck/types.py", line 82, in from_raw_script
    output = get_output(script, expanded)
  File "/usr/lib/python3.7/site-packages/thefuck/output_readers/__init__.py", line 20, in get_output
    return rerun.get_output(script, expanded)
  File "/usr/lib/python3.7/site-packages/thefuck/output_readers/rerun.py", line 56, in get_output
    is_slow = shlex.split(expanded) in settings.slow_commands
  File "/usr/lib/python3.7/shlex.py", line 305, in split
    return list(lex)
  File "/usr/lib/python3.7/shlex.py", line 295, in __next__
    token = self.get_token()
  File "/usr/lib/python3.7/shlex.py", line 105, in get_token
    raw = self.read_token()
  File "/usr/lib/python3.7/shlex.py", line 206, in read_token
    raise ValueError("No escaped character")
ValueError: No escaped character
```

**Comment by alecastle** [comment]
To be clear, the behavior I'm expecting is that after having run `sudo pacman -s atom`, running `fuck` would correct it to `sudo pacman -S atom`

**Comment by AriEythorsson** [comment]
I'm on it

**Comment by scorphus** [comment]
Duplicate of #994 which was fixed in c2df71c.
